zerver/lib/actions.py

```diff
@@ -1813,6 +1816,7 @@ def build_message_send_dict(
         online_push_user_ids=info["online_push_user_ids"],
         stream_push_user_ids=info["stream_push_user_ids"],
         stream_email_user_ids=info["stream_email_user_ids"],
+        sender_muting_user_ids=info["sender_muting_user_ids"],
```

Comment:
I think `muted_sender_user_ids` would feel more readable to me, reading as "users who muted the sender".  And a local variable of `muted_sender` from that makes sense as well.